src/wallet/ledger/ledger.ts

```diff
@@ -44,6 +44,10 @@ export class LedgerSigner implements Signer {
     return this.connector.getPubkey(this.hdPath);
   };
 
+  getPrivateKey = async (): Promise<Uint8Array> => {
+    throw new Error('Ledger does not support private key');
```

Comment:
We should expand this message to be:

`Ledger does not support private key exports`